pages/common/xkill.md

```diff
@@ -2,6 +2,7 @@
 
 > Kill a window interactively in a graphical session.
 > See also `kill` and `killall`.
+> More information: <https://man.openbsd.org/xkill>.
```

Comment:
Why did you use an `openbsd` man page here. `xkill` wasn't developed by `openbsd` right?
Same with `Xeypher` and `xargs`

Reply:
Sorry about this, learnt something new today.

I've now updated the links for:
* xkill - https://www.x.org/releases/current/doc/man/man1/xkill.1.xhtml
* xepher - https://cgit.freedesktop.org/xorg/xserver/plain/hw/kdrive/ephyr/README
* xargs - https://pubs.opengroup.org/onlinepubs/9699919799/utilities/xargs.html

Reply:
`xkill` and `xargs` look good, but I think we should use <https://manned.org/xserver-xephyr> for `xephyr`

Reply:
Thanks @marchersimon for the suggestion, have updated `xephyr` link.